### DISCUSSION TITLE
Issue with double screen

### DISCUSSION THREAD

**Discussion by Domx93** [flameshot-org/flameshot#3385] [Q&A] [answered]
Hello, I have a problem when I try to use flameshot with two screen connected, it's like the two screen are shifted, I see a part of the main screen on the second one and I lose a part of the second one. I attach two picture for better understanding. I use the lastest archlinux with plasma (Xorg), but I have this problem since months. The command "flameshot full" works well.

![photo-2023-10-19-15-49-17](https://github.com/flameshot-org/flameshot/assets/30021637/3dc1cfa7-7530-4a99-828e-6122a471a85c)
![photo-2023-10-19-15-49-12](https://github.com/flameshot-org/flameshot/assets/30021637/ddb1559d-9a61-47e5-bee3-1ad16bd8a978)

**Comment by mmahmoudian** [comment, marked as answer]
Would you please attach the pictures inside the Github instead of uploading somewhere else and make a shortened url? 


Also read the following issue and see if you have the same setup (i.e fractional scaling)

https://github.com/flameshot-org/flameshot/issues/3180

**Reply by Domx93** [reply]
I didn't find this topic, anyway It solved the issue, thank you! :D
